### DISCUSSION TITLE
Caused by: java.lang.ClassNotFoundException: org.apache.spark.shuffle.sort.ColumnarShuffleManager

### DISCUSSION THREAD

**Discussion by njalan** [apache/gluten#2225] [General]
Below is my config:
spark-shell --master yarn --deploy-mode client --conf spark.driver.extraClassPath=/data/gluten-velox-bundle-spark3.2_2.12-centos_7-0.5.0-SNAPSHOT.jar,/data/gluten-thirdparty-lib-centos-7.jar --conf spark.executor.extraClassPath=/data/gluten-velox-bundle-spark3.2_2.12-centos_7-0.5.0-SNAPSHOT.jar,/data/gluten-thirdparty-lib-centos-7.jar --conf spark.plugins=io.glutenproject.GlutenPlugin --conf spark.gluten.sql.columnar.backend.lib=velox --conf spark.gluten.loadLibFromJar=true --conf spark.shuffle.manager=org.apache.spark.shuffle.sort.ColumnarShuffleManager --conf spark.memory.offHeap.size=10G --queue batch --jars /data/gluten-velox-bundle-spark3.2_2.12-centos_7-0.5.0-SNAPSHOT.jar,/data/gluten-thirdparty-lib-centos-7.jar
I faced below error messages:
Caused by: java.lang.ClassNotFoundException: org.apache.spark.shuffle.sort.ColumnarShuffleManager

**Comment by philo-he** [comment]
Hi @njalan, it looks the jars were wrongly configured for `spark.driver.extraClassPath` & `spark.executor.extraClassPath`. Please separate the two configured jars with ":" instead of ",".
BTW, could you tell us your company name? Thanks!
